device.yml

```diff
@@ -0,0 +1,127 @@
+%YAML 1.1
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/harp-tech/reflex-generator/main/schema/device.json
+device: Rfid
+whoAmI: 2094
+firmwareVersion: "0.2"
+hardwareTargets: "1.2"
+registers:
+  InboundDetectionId: &detectionevent
+    address: 32
+    type: U64
+    access: Event
+    description: The ID of the tag that was detected as in entered the area of the reader.
+  OutboundDetectionId:
+    << : *detectionevent
+    address: 33
+    description: The ID of the tag that was detected as in exited the area of the reader.
+  DO0State:
+    address: 34
+    type: U8
+    access: [Write, Event]
+    maskType: DigitalOutputState
+    description: Changes the state of the digital output pin.
+  EnableHardwareNotifications:
+    address: 35
+    type: U8
+    access: Write
+    maskType: HardwareNotifications
+    description: Enables or disables hardware notifications.
+  TriggerHardwareNotifications:
+    address: 36
+    type: U8
+    access: Write
+    maskType: HardwareNotifications
+    description: Triggers hardware notifications.
+  BuzzerNotificationDuration: &notificationduration
+    address: 37
+    type: U16
+    access: Write
+    minValue: 2
+    description: The time the buzzer will stay on (sensitive to multiples of 2).
+  TopLedNotificationDuration:
+    << : *notificationduration
+    address: 38
+    description: The time the top LED will stay on (sensitive to multiples of 2).
+  BottomLedNotificationDuration:
+    << : *notificationduration
+    address: 39
+    description: The time the bottom LED will stay on (sensitive to multiples of 2).
+  BuzzerNotificationFrequency:
+    address: 40
+    type: U16
+    access: Write
+    minValue: 200
+    maxValue: 15000
+    description: The frequency of the buzzer notification.
+  TopLedNotificationPeriod: &notificationblinkperiod
+    address: 41
+    type: U16
+    access: Write
+    minValue: 2
+    description: The blink period of the top LED notification (sensitive to multiples of 2).
+  BottomLedNotificationPeriod:
+    << : *notificationblinkperiod
+    address: 42
+    description: The blink period of the bottom LED notification (sensitive to multiples of 2).
+  Reserved0: &reserved
+    address: 43
+    type: U8
+    access: Read
+    visibility: private
+    description: Reserved for future use.
+  Reserved1:
+    << : *reserved
+    address: 44
+    description: Reserved for future use.
+  MatchTagId0: &matchtagid #TODO not sure what event this triggers...
```

Comment:
@filcarv It is not clear what event is triggered here since this event is write only...

Reply:
@bruno-f-cruz I was just discussing this with @filcarv and these registers are actually used as filters to allow only specific IDs to trigger the `InboundDetectionId` event, i.e. if any of these registers is written, then only IDs in the list will trigger the event. By default, all IDs are allowed.